core/lib/config/src/configs/chain.rs

```diff
@@ -123,7 +127,7 @@ pub struct StateKeeperConfig {
 
     /// Number of keys that is processed by enum_index migration in State Keeper each L1 batch.
     pub enum_index_migration_chunk_size: Option<usize>,
-
+    #[serde(default = "default_l1_batch_commit_data_generator_mode")]
```

Comment:
I think we can get rid of `default_l1_batch_commit_data_generator_mode` and set `    L1BatchCommitDataGeneratorMode::Rollup` directly (let me know if this doesn't work)

```suggestion
    #[serde(default = "default_l1_batch_commit_data_generator_mode")]
```

Reply:
I've researched a little bit and it seems that given that `L1BatchCommitDataGeneratorMode` implements `Default` (as `Rollup`) you can just do `#serde(default)`.

Src: https://serde.rs/attr-default.html